README.md

```diff
@@ -130,6 +130,8 @@ example_hex_flow()
 
 ## Resources
 
+Read [this article](https://towardsdatascience.com/create-observable-and-reproducible-notebooks-with-hex-460e75818a09) or watch [this video](https://youtu.be/_BjqCrun4nE) to learn more about prefect-hex.
```

Comment:
I think this formatting might encourage more people to click through, but let me know what you think.

```suggestion
### Blog Posts

- [Create Observable and Reproducible Notebooks with Hex](https://towardsdatascience.com/create-observable-and-reproducible-notebooks-with-hex-460e75818a09) by Khuyen Tran

### Videos

- [Create Observable and Reproducible Notebooks with Hex: Why Hex (Part 1)](https://youtu.be/_BjqCrun4nE)
```

Reply:
It looks great to me